interpretation/ww_head.py

```diff
@@ -212,8 +212,16 @@ def process(message, isGameMaster = False):
         # This command removes a given user from a conspiracy channel.
         # A user should not get removed if they're the channel owner.
         if is_command(message,['remove']):
-            # TODO
-            return todo()
+            members_to_remove = check.users(message)
+            if is_owner(user_id,channel_id) == False:
+                return [Mailbox().respond("I\'m sorry, but you cannot use this command over here!")]
+            command = Mailbox()
+            for x in members_to_remove:
+                if is_owner(x,channel_id) == True:
+                    return [Mailbox().respond("The owner of a CC can\'t be removed! Please try again.")]
+                command.edit_cc(x,channel_id,0)
```

Comment:
We need to think about this. Do we want players to explicitly disallow players access to the conspiracy channel?

By default, spectators and dead players automatically gain access to conspiracy channels. I can imagine one not wanting to allow a dead player access if they suspect them of cheating, if they do not want that player to see the conspiracy channel or to hide ghosts from appearing on the list.

If we want to allow that, this is just fine. For now, let's merge it anyway. But it is something we need to think about.
Is there an easy way to make an issue out of this post?